resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,117 @@
+![alt text](../../.gitbook/assets/push_token_to_identity.png "Verify SetPushIdentifier event.")
+   2. Verify that app’s push token is in the Campaign instance that was mapped with the correct Experience cloud id(mid) by clicking _Adobe Campaign -> Administrator -> Channels -> Mobile App (AEP SDK)_.
+   3. Launch your app and, under the mobile application subscribers, review the list of all Subscribers of app.
+   4. Verify the Experience Cloud ID and Registration Token for the user is displayed.
+   
+   ![alt text](../../.gitbook/assets/campaign_app_subscriber_list.png "App subscriber list, verify mid and push token.")
+   
+   5. If you are using Charles, verify that the push token is successfully synced with the ECID service. Check for the **demdex request** which is marked with the red line in the screenshot below. Also verify the successfull resposne(200) for this network call.  
+ 
+ ![alt text](../../.gitbook/assets/push_identifier.png "ECID network request for push token sync.")
+ 
+ 
+ ## Troubleshooting push notification events tracking  
+
+The following events are related to the tracked push notifications:
+  * Impression (Notification is delivered.)
+  * Click (User clicked the notification.)
+  * Open (App opened in response to user's click on notification.)  
+  For more information about implementing the tracking of push notification events, see [Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).  
+  
+###  Troubleshooting using Project Griffon  
+
+  You can Verify push token sync with the Experience Cloud Identity service on Project Griffon.  
+  Launch your app with the device connected to a Project Griffon session. Receive a push notification in the device.  
+  In the list of events, verify that you have an events with type CollectData. In the details panel on the right, verify the value of the action. The value action should be 7 for impression, 2 for click and 1 for open. It is shown in the screenshots below.  
+  
+   ![alt text](../../.gitbook/assets/push_tracking_impression.png "Shows Push notification impression tracking.")
+   
+   Shows tracking of **impression** event. It is indicated by action 7.
+   
+   ![alt text](../../.gitbook/assets/push_tracking_click.png "Shows Push notification click tracking.")
+   
+   Shows tracking of **click** event. It is indicated by action 2.
+   
+   ![alt text](../../.gitbook/assets/push_tracking_open.png "Shows Push notification open tracking.")
+   
+   Shows tracking of **open** event. It is indicated by action 1.
+  
+  
+###  Troubleshooting using Charles  
+
+To verify that a successfull network call is made to track the push notification interaction event, look for GET request with query string `id=broadlogID,deliveryID,action` to campaign server, see screenshots below for mopre details about what to look.  
+
+**_Screenshot for the impression event tracking(action value 7)_**
```

Comment:
Too many screenshots again...please see my earlier comment about possible showing the 3 pieces of information in 1 screenshot and removing the unnecessary ones.